.ci/docker/common/install_onnx.sh

```diff
@@ -23,3 +23,18 @@ pip_install \
 
 # TODO: change this when onnx-script is on testPypi
 pip_install "onnx-script@git+https://github.com/microsoft/onnx-script@29241e15f5182be1384f1cf6ba203d7e2e125196"
+
+IMPORT_SCRIPT_FILENAME="/tmp/onnx_import_script.py"
+# Cache the transfomers model to be used later by ONNX tests. By default, the model
```

Comment:
This section could use a few more comments, since it would be really hard to figure out what this is doing outside the context of this pull request.

Maybe something like:

```suggestion
# Cache the transformers model to be used later by ONNX tests. We need to run the transformers python package to download the model

By default, the model
```